dotcom-rendering/src/web/components/PinnedPost.tsx

```diff
@@ -140,6 +140,7 @@ export const PinnedPost = ({ pinnedPost, children, format }: Props) => {
 	const palette = decidePalette(format);
 	return (
 		<div
+			id="pinned-post"
```

Comment:
Do you know if the sharing links (Facebook and Twitter) still work ok? They take you to the pinned post at the top at the moment though one could argue they should take you to the original post